request_logger.py

```
@@ -0,0 +1,105 @@
+            'elapsed': str(response.elapsed),
+        }
+        
+        # Add response headers
+        response_data['headers'] = dict(response.headers)
+        
+        # Try to add response content for reasonable-sized responses
+        try:
+            if int(response.headers.get('content-length', 0)) < 10000:
+                try:
+                    response_data['content'] = response.json()
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)